lib/ansible/modules/inventory/add_host.py

```diff
@@ -1,81 +1,83 @@
 # -*- mode: python -*-
-#
+
+# Copyright: (c) 2012, Seth Vidal (@skvidal)
 # Copyright: Ansible Team
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['stableinterface'],
                     'supported_by': 'core'}
 
-
 DOCUMENTATION = r'''
 ---
 module: add_host
-short_description: add a host (and alternatively a group) to the ansible-playbook in-memory inventory
+short_description: Add a host (and alternatively a group) to the ansible-playbook in-memory inventory
 description:
-  - Use variables to create new hosts and groups in inventory for use in later plays of the same playbook.
-    Takes variables so you can define the new hosts more fully.
-  - This module is also supported for Windows targets.
+- Use variables to create new hosts and groups in inventory for use in later plays of the same playbook.
+- Takes variables so you can define the new hosts more fully.
+- This module is also supported for Windows targets.
 version_added: "0.9"
 options:
   name:
-    aliases: [ 'hostname', 'host' ]
     description:
     - The hostname/ip of the host to add to the inventory, can include a colon and a port number.
+    type: str
     required: true
+    aliases: [ host, hostname ]
   groups:
-    aliases: [ 'groupname', 'group' ]
     description:
-    - The groups to add the hostname to, comma separated.
-    required: false
+    - The groups to add the hostname to.
+    type: list
+    aliases: [ group, groupname ]
 notes:
-    - This module bypasses the play host loop and only runs once for all the hosts in the play, if you need it
-      to iterate use a with\_ directive.
-    - Windows targets are supported by this module.
-    - The alias 'host' of the parameter 'name' is only available on >=2.4
-    - Since Ansible version 2.4, the ``inventory_dir`` variable is now set to ``None`` instead of the 'global inventory source',
-      because you can now have multiple sources.  An example was added that shows how to partially restore the previous behaviour.
+- This module bypasses the play host loop and only runs once for all the hosts in the play, if you need it
+  to iterate use a with\_ directive.
```

Comment:
```suggestion
  to iterate use a C(with_) directive.
```

Reply:
Well, it's not exactly a value. I'd rather say with-loop or something like that.